src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/domain/TierCalculation.kt

```diff
@@ -146,16 +145,34 @@ class TierCalculation {
 
     private fun getComplexityPoints(
       complexityFactors: List<ComplexityFactor>,
-      assessmentComplexityFactors: List<AssessmentComplexityFactor>
+      assessmentComplexityFactors: Map<AssessmentComplexityFactor, String?>
     ): Int {
+
       return if (complexityFactors.any() || assessmentComplexityFactors.any()) {
         criteria.add(TierMatchCriteria.INCLUDED_COMPLEXITY_FACTORS)
-        complexityFactors.distinct().count().plus(assessmentComplexityFactors.distinct().count()) * 2
+        (complexityFactors.distinct().count().plus(getAssessmentComplexityPoints(assessmentComplexityFactors))) * 2
       } else {
         criteria.add(TierMatchCriteria.NO_COMPLEXITY_FACTORS)
         0
       }
     }
+
+    private fun getAssessmentComplexityPoints(factors: Map<AssessmentComplexityFactor, String?>): Int {
+
+      val parentingAnswer = factors.getOrDefault(AssessmentComplexityFactor.PARENTING_RESPONSIBILITIES, "N")
+      val parenting = if (isYes(parentingAnswer)) 1 else 0
+
+      // We dont take the cumulative score, just '1' if at least one of these two is present
+      val impulsivityAnswer = factors[AssessmentComplexityFactor.IMPULSIVITY]?.toInt() ?: 0
+      val temperControlAnswer = factors.getOrDefault(AssessmentComplexityFactor.TEMPER_CONTROL, "0")?.toInt() ?: 0
+      val selfControl = if (impulsivityAnswer.plus(temperControlAnswer) > 1) 1 else 0
```

Comment:
Should this be > 0 rather than > 1?
" if they have one or the other or both"

Reply:
it should be, good spot, its because in the tests I've used 1 and 2 as the scores, would have caught it if i'd used 0 and 1